src/app/Processor/ProcessorEdit/useValidateConfigParams.ts

```diff
@@ -59,7 +59,9 @@ export function useValidateConfigParams(
         if (field) {
           // Run the field validation function provided by configTypes with the current
           // value of the field
-          const fieldValidation = field.validate(config.parameters[fieldName]);
+          const fieldValidation = field.validate(
+            config.parameters[fieldName] as string
+          );
```

Comment:
where do we have validate implementation please? I think we need now in that implementation check what is the type of the passed parameter? is that actually implemented on backend?